bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/game/Game.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * File: game.java
+ * Class: Game
+ *
+ * Version: 0.0.1
+ * Date: September 8th, 2016
+ */
+
+package nl.tudelft.bust_a_move20162017.bust_a_move_framework.game;
+
+import nl.tudelft.bust_a_move20162017.bust_a_move_framework.cannon.Cannon;
+import nl.tudelft.bust_a_move20162017.bust_a_move_framework.arena.Arena;
+import nl.tudelft.bust_a_move20162017.bust_a_move_framework.player.Player;
+import java.awt.Color;
+import java.awt.Graphics;
+
+/**
+ * The Game class represents a game entity.
+ *
+ * @author Maurice Willemsen
+ */
+
+public class Game {
+
+	private enum State {
+		START, PLAYING, FAILED, WON
+	}
+
```

Comment:
Try to keep relate variables together. In other words, without an empty line between each variables